src/main/java/trello/ui/pages/BoardPage.java

```diff
@@ -26,16 +26,16 @@
 public class BoardPage extends BasePage {
 
     @FindBy(css = ".placeholder")
```

Comment:
Replace css by className for find WebElement please.

Reply:
done.